sbin/spark-daemon.sh

```diff
@@ -164,9 +165,10 @@ case $option in
   (stop)
 
     if [ -f $pid ]; then
-      if kill -0 `cat $pid` > /dev/null 2>&1; then
+      TARGET_ID=`cat $pid`
```

Comment:
Same feedback as [the identical line above](https://github.com/apache/spark/pull/3683/files#diff-92634b2c881c4bf504852e7b963e7894R132).